lab_1_keywords_tfidf/main.py

```diff
@@ -137,7 +227,19 @@ def calculate_chi_values(expected: dict[str, float], observed: dict[str, int]) -
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if expected and isinstance(expected, dict) and observed and isinstance(observed, dict):
```

Comment:
revert the condition and return None immediately

Reply:
Done